### DISCUSSION TITLE
[vcpkg upgrade] How do I upgrade packages of specific triplet ?

### DISCUSSION THREAD

**Discussion by brukted** [microsoft/vcpkg#19137] [Ideas]
I want  to upgrade packages I installed for a specific triplet (x64-windows). So far I have tried `` vcpkg upgrade --triplet x64-windows `` but It is the same as ``vcpkg upgrade`` .

**Comment by cenit** [comment]
Unfortunately you have to specify ports one by one, each one with its own triplet next to it, on the command line
If you want to contribute a better experience, I think that many people would be very happy :) 

`./vcpkg upgrade bzip2:x64-windows libzip:x64-windows` ...
